### PR TITLE
Hugo 15 support Markdown per GitHub; Removing seds

### PR DESCRIPTION
**Pull request by moxiegirl**
All the links reverted to GitHub Style.

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -61,21 +61,21 @@ The top of each Docker Compose documentation file contains TOML metadata. The me
 The metadata alone has this structure:
 
     +++
-    title = "Extending services in Compose"
-    description = "How to use Docker Compose's extends keyword to share configuration between files and projects"
-    keywords = ["fig, composition, compose, docker, orchestration, documentation, docs"]
+    title = "Extending services in machine"
+    description = "How to use Docker machine's extends keyword to share configuration between files and projects"
+    keywords = ["fig, composition, machine, docker, orchestration, documentation, docs"]
```
Comment (thaJeztah) [inline docs/README.md]: Is this correct? seems like a copy/paste/find/replace error

Comment (thaJeztah) [inline docs/README.md]: O, wait its just to illustrate things :smile:

Comment (moxiegirl) [inline docs/README.md]: yep

**Comment by thaJeztah** [comment]
LGTM

**Comment by nathanleclaire** [comment]
LGTM

**Comment by moxiegirl** [comment]
Cool. Ty

**Pull request merged**